lib/gracenote/client.rb

```diff
@@ -24,6 +26,10 @@ def lang=(lang)
       @lang = lang
     end
 
+    def country=(country)
```

Comment:
__[rubocop]__
* Use `attr_writer` to define trivial writer methods. [:ref](https://github.com/bbatsov/ruby-style-guide#attr_family)